### PR TITLE
225 create a development dataset

### PR DESCRIPTION
**Pull request by rnudb**

### CODE REVIEW

**Comment by ArnaudChirat** [comment]
The CHANGELOG and the README must be update

**Review by ArnaudChirat — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# v1.16.0 (2022-10-27)
```
Comment (bport) [inline CHANGELOG.md:1]: mettre xxxx-xx-xx à la palce de la date on a pas encore de date de release

File: interview/management/commands/create_dev_dataset.py [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+
+def get_available_consultants_for_itw(subsidiary, all_itw_given_process):
+    # retrieve available consultants that have not yet been involved in the process
+    possible_interviewer = (
+        Consultant.objects.filter(subsidiary=subsidiary)
+        .exclude(id__in=all_itw_given_process.values_list("interviewers", flat=True))
+        .distinct()
+    )
+    # if all consultants were already involved in the process, choose one at random
+    if len(possible_interviewer) == 0:
```
Comment (bport) [inline interview/management/commands/create_dev_dataset.py:100]: We can write directly

if possible_interviewer:

File: interview/factory.py [hunk 3]
```diff
@@ -1,34 +1,137 @@
-import factory
-import factory.faker
-import factory.fuzzy
-
 import datetime
+import random
```
Comment (bport) [inline interview/factory.py:2]: don't use random we want the ability in future to have something seeded and use faker everywhere will ease that

File: interview/management/commands/create_dev_dataset.py [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+
+    return possible_interviewer
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        pass
+
+    def handle(self, *args, **options):
+        # call_command("flush", "--no-input") # dev
```
Comment (bport) [inline interview/management/commands/create_dev_dataset.py:111]: This comment need to go to readme I guess

File: interview/management/commands/create_dev_dataset.py [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+For now 2 subsidiaries
+            5 users per subsidiary
+            around 100 process per subsidiary in the last 2 years
+                an average of 3 itw per process
+"""
+
+
+def generate_basic_data(subsidiary):
+    # generate InterviewKind
+    if InterviewKind.objects.all().count() == 0:
```
Comment (bport) [inline interview/management/commands/create_dev_dataset.py:39]: Why are we checking that ? (apply to same stuff below)
If we wanted to check that we can use .`exists()`

File: interview/management/commands/create_dev_dataset.py [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+        for i in range(1, 5):
+            InterviewKindFactory(name="Interview Kind {no}".format(no=i))
```
Comment (bport) [inline interview/management/commands/create_dev_dataset.py:41]: I think the factory need to generate random name by it self and we don't need to put extra argument there

**Review by bport — CHANGES_REQUESTED** [review]

**Pull request merged**